website/static/js/fangorn.js

```diff
@@ -1609,14 +1613,37 @@ function expandStateLoad(item) {
             }
         }
     }
+
     if (item.children.length > 0 && item.depth === 2) {
         for (i = 0; i < item.children.length; i++) {
             if (item.children[i].data.isAddonRoot || item.children[i].data.addonFullName === 'OSF Storage' ) {
                 tb.updateFolder(null, item.children[i]);
             }
         }
     }
-        $('.fangorn-toolbar-icon').tooltip();
+
+    if (item.depth > 2 && !item.data.isAddonRoot && item.children.length === 0 && item.open) {
+        if (icon.get(0)) {
+            m.render(icon.get(0), tbOptions.resolveRefreshIcon());
+        }
+        $osf.ajaxJSON(
+            'GET',
+            '/api/v1/project/' + item.data.nodeID + '/files/grid/'
+        ).done(function(xhr) {
```

Comment:
"xhr" is misleading. The first argument is not an `XHR` object, it's the parsed response data. Maybe "response" or "resp"?